test/behaviors/conference.behavior.js

```diff
@@ -533,6 +532,121 @@ function shouldBehaveLikeConference () {
       assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(deposit, 0))
     })
   })
+
+  describe('on clear and send', function(){
+    let conference, deposit, attended, payoutAmount, clearFee, fees;
+
+    beforeEach(async function(){
+      conference = await createConference({coolingPeriod:0})
+      deposit = await conference.deposit()
+      
+      let numRegistered = 4;
+      for (let i = 0; i < numRegistered; i++) {
+          await register({conference, deposit, user:accounts[10 + i], owner})
+      } 
+
+      // [ 110 ], accounts[11], accounts[12]
+      const maps = [ toBN(0).bincn(1).bincn(2) ];
+
+      await conference.finalize(maps, {from:owner});
+      await conference.ended().should.eventually.eq(true)
+      
+      payoutAmount = new EthVal(await conference.payoutAmount());
+      clearFee = await conference.clearFee();
+      fees = payoutAmount.mul(clearFee).div(1000).toString(10);
+
+      assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(deposit, 4))
```

Comment:
4 can be replaced with `numRegistered` to remove the magic number

Reply:
Okay, got it. Replaced.